1-Draft/RFCXXXX-RFC-Invoke-DscResource.md

```diff
@@ -90,32 +103,114 @@ Invoke-DscResource -Name Script -ModuleName @{ModuleName='PSDscResources';Module
 
 ```
 
-What will be executed will be equivalent (in terms of scope) to:
-
-```PowerShell
-Start-Job -Credential $Credential -ScriptBlock {
-    Invoke-DscResource -Name Script -ModuleName @{ModuleName='PSDscResources';ModuleVersion='2.12.0.0'} -Method 'Set' -Properties @{
-        GetScript  = '<# My Get ScriptBlock #>'
-        SetScript  = '<# My Set ScriptBlock #>'
-        TestScript = '<# My Test ScriptBlock #>'
-    } -Verbose
-}
-```
+Will **throw an exception**, because of the `PsDscRunAsCredential` key.
 
 > NOTES: We're avoiding to take dependencies on other technologies that would require extra configurations or permissions (such as remoting), or would not be available on other OSes.
 
 #### Independent and isolated execution
 
-`Invoke-DscResource` in PowerShell 7+ will not be aware of other instances being executed, and as such it will be possible to execute several instances in parallel when isolated in their own runspaces, or run in parallel with the LCM.
+`Invoke-DscResource` in PowerShell 7+ will not be aware of other instances being executed, and as such it will be possible to execute several instances in parallel when isolated in their own runspace, or run in parallel with the LCM.
 
 This means that it enables concurrent execution, but also risks conflict if two conflicting resources are run simultaneously.
 
 It is up to the user to sequence the execution safely, or to create appropriate resources.
 
+#### Output & Types
+
+##### Get
+
+The `GET` function in DSC resources returns a Hashtable, and this will be the same
```

Comment:
the LCM version return a CIM representation of a hashtable.  We will return the actual hashtable.

Reply:
fixed